### DISCUSSION TITLE
Cant use wear os version of Capod

### DISCUSSION THREAD

**Discussion by nilo19-yt** [d4rken-org/capod#272] [Q&A]
So basically when I try to use the app on my wear os it's asks for permission and everything goes well until the notification permission whenever I tap on grant permission it just can't And I tried on setting permission manager but I couldn't find a notification permission for the app

**Comment by Macusercom** [comment]
https://github.com/d4rken-org/capod/issues/154

Might be fixed here: https://github.com/d4rken-org/capod/pull/203
